dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7827,22 +7261,6 @@ public String toString() {
           .setScope(Scope.ALL)
           .build();
 
-  public static final PropertyKey DORA_UFS_FILE_STATUS_CACHE_SIZE =
-      intBuilder(Name.DORA_UFS_FILE_STATUS_CACHE_SIZE)
-          .setDefaultValue(100000)
-          .setDescription("The max size of the cache of UFS file status")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
-          .setScope(Scope.WORKER)
-          .build();
-
-  public static final PropertyKey DORA_UFS_FILE_STATUS_CACHE_TTL =
-      durationBuilder(Name.DORA_UFS_FILE_STATUS_CACHE_TTL)
-          .setDefaultValue("48h")
-          .setDescription("The TTL of the cache of UFS file status")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
-          .setScope(Scope.WORKER)
-          .build();
```

Comment:
do we wanna keep the dora related ones? Could they be used later?

Reply:
it's for the primitive stage dora, I think we don't use them any more.